docs/5.x/changelog.md

```diff
@@ -22,6 +23,21 @@ LTS starts with `3.51.0` with minor backwards compatible changes added over time
 
 ## 5.x Releases
 
+### 5.3.0
+
+#### Improvements
+
+* Upgrade to Kubernetes `1.12.1`.
+* Replace `kube-dns` with CoreDNS.
+* Remove dependency on system user/group being present in local `/etc/passwd` and `/etc/group` databases.
+
+### 5.2.1 LTS
+
+#### Bugfixes
+
+* Fix an issue with garbage collection refusing to launch on active clusters.
```

Comment:
This was not an issue for the user-facing `gravity gc` but low-level `system gc` so I'm not sure this needs to be advertised.